bot/exts/moderation/slowmode.py

```diff
@@ -26,8 +30,15 @@
 class Slowmode(Cog):
     """Commands for getting and setting slowmode delays of text channels."""
 
+    # Stores the expiration timestamp in POSIX format for active slowmodes, keyed by channel ID.
+    slowmode_expiration_cache = RedisCache()
+
+    # Stores the original slowmode interval by channel ID, allowing its restoration after temporary slowmode expires.
+    original_slowmode_cache = RedisCache()
```

Comment:
Given these are both structured and known values, I would maybe consider serializing them together into one store?

This is a nit though and others may disagree, it just appears that we use them more or less in the same place so whilst if we were just working in dict-land this would be preferred when working in a place with an underlying store I prefer it being the same.